src/ngAria/aria.js

```diff
@@ -235,30 +251,24 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
           }
         },
         post: function(scope, elem, attr, ngModel) {
-          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem)
-                                && !isNodeOneOf(elem, nodeBlackList);
+          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem, false);
+          var ngTrueValue = $parse(attr.ngTrueValue)(scope);
 
           function ngAriaWatchModelValue() {
             return ngModel.$modelValue;
           }
 
           function getRadioReaction() {
-            if (needsTabIndex) {
-              needsTabIndex = false;
-              return function ngAriaRadioReaction(newVal) {
-                var boolVal = (attr.value == ngModel.$viewValue);
-                elem.attr('aria-checked', boolVal);
-                elem.attr('tabindex', 0 - !boolVal);
-              };
-            } else {
-              return function ngAriaRadioReaction(newVal) {
-                elem.attr('aria-checked', (attr.value == ngModel.$viewValue));
-              };
-            }
+            return function ngAriaRadioReaction(newVal) {
+              var boolVal = (attr.value == ngModel.$viewValue);
+              elem.attr('aria-checked', boolVal);
+            };
           }
-
-          function ngAriaCheckboxReaction() {
-            elem.attr('aria-checked', !ngModel.$isEmpty(ngModel.$viewValue));
+          function getCheckboxReaction() {
```

Comment:
It was added to be consumed in a watch function on [line 282](https://github.com/angular/angular.js/pull/13483/files#diff-91e9129201746ba107348c9a0a7735edR282).